### DISCUSSION TITLE
How to update the grid with each step of displacement？

### DISCUSSION THREAD

**Discussion by lyubing0716** [Ferrite-FEM/Ferrite.jl#1027] [Q&A]
Under the updated Lagrangian framework, I want to achieve the coordinates of each node in a grid change with node displacements. But for a 'grid', the coordinates of its' nodes' cannot be directly modified, I can only create a Node((new_Coord[1], new_Coord[2])) to assign to the original Node, Is there any better way to achieve this function?

**Comment by KristofferC** [comment]
There is the `transform_coordinates!` function.
